rmf_demos_fleet_adapter/rmf_demos_fleet_adapter/RobotCommandHandle.py

```diff
@@ -189,6 +182,12 @@ def follow_new_path(
             path_finished_callback):
 
         self.stop()
```

Comment:
With https://github.com/open-rmf/rmf_ros2/pull/167, do we still need to call `self.stop()` here?

Reply:
You're right we don't, especially with demos' stop endpoint. Updated accordingly